iosApp/ci_scripts/ci_post_clone.sh

```diff
@@ -63,6 +63,7 @@ retry ./gradlew :shared:generateDummyFramework
 cd "${CI_PRIMARY_REPOSITORY_PATH}/iosApp"
 retry bundle exec pod install
 cd ..
+retry ./gradlew :shared:podInstallSyntheticIos
```

Comment:
Hmm, this seems to be failing because it can't find the pod install
```
Execution failed for task ':shared:podInstallSyntheticIos'.

>         ERROR: CocoaPods executable not found in your PATH.

          Please make sure CocoaPods is installed on your system.
```
Which is strange, since you can see the pod install complete directly above it. Could XCC be messing with the path somehow? Or could the dependency license changes have messed up gradle's path?

Reply:
When we install CocoaPods via Bundler, it's not actually on the `PATH`, and `bundle exec` rewrites the `PATH` to include all the gems it manages. It might be sufficient to put the Gradle task inside `bundle exec`, though.